examples/eSealing/src/eSealing.tsx

```diff
@@ -0,0 +1,435 @@
+                                checked={!isRegisterFilePage}
+                                checkedIcon={false}
+                                uncheckedIcon={false}
+                            />
+                            <button
+                                style={isRegisterFilePage ? ButtonStyleNotSelected : ButtonStyleSelected}
+                                type="button"
+                                onClick={() => {
+                                    setIsRegisterFilePage(false);
+                                    setFileHashHex('');
+                                    setWitness('');
+                                    setTimestamp('');
+                                    setHash('');
+                                }}
+                            >
+                                Display Tab
+                            </button>
+                        </div>
+                    </>
+                )}
+                {activeConnectionGenesisHash && activeConnectionGenesisHash !== network.genesisHash && (
+                    <p style={{ color: 'red' }}>
+                        Unexpected genesis hash: Please ensure that your wallet is connected to network{' '}
+                        <code>{network.name}</code>.
+                    </p>
+                )}
+            </div>
+
+            <label>
+                {activeConnectedAccount !== undefined && (
+                    <>
+                        <p style={{ marginBottom: 0 }}>Select a file:</p>
+                        <input
+                            className="input"
+                            style={InputFieldStyle}
+                            type="file"
+                            onChange={changeHandler}
+                            ref={file}
+                        />
+                        <button
+                            style={ButtonStyle}
+                            type="button"
+                            onClick={async () => {
+                                try {
+                                    if (selectedFile !== undefined) {
+                                        setFileHashHex('');
+                                        setLoading(true);
+                                        const arrayBuffer = await selectedFile.arrayBuffer();
+                                        const byteArray = new Uint8Array(arrayBuffer as ArrayBuffer);
+                                        const newFileHashHex = sha256(byteArray.toString());
+                                        setFileHashHex(newFileHashHex);
+                                        setLoading(false);
+                                    } else {
+                                        alert('Choose a file to compute the file hash');
+                                    }
+                                } catch (err) {
+                                    console.error(setLoadingError((err as Error).message));
```

Comment:
```suggestion
                                    setLoadingError((err as Error).message);
```

Setter returns void.